### PR TITLE
feat: Workflow aligned with "apps" and "service" WL structures

### PR DESCRIPTION
**Pull request by vtsarfin-cg**

### CODE REVIEW

File: .github/workflows/multi_service_image_build.yml [hunk 1]
```diff
@@ -54,32 +58,53 @@ jobs:
...
           echo "CHANGED workload-services (array): ${CHANGED_WLS_DIRS[@]}"
           if [[ ${#CHANGED_WLS_DIRS[@]} -gt 0 ]]
           then
-           argo submit .argo/version-changer-wf.yaml \
-              -p gitops-repo=git@github.com:<GIT_ORGANIZATION_NAME>/<WL_GITOPS_REPO_NAME>.git \
+             echo "${CHANGED_WLS_DIRS[@]}"
+             if  [[ $APP_STRUCT -ne 1 ]]
+             then
+               argo submit .argo/version-changer-wf.yaml \
+              -p gitops-repo=git@github.com:cgdevx-demo/vt-cnask-gitops.git \
```
Comment (sergs-pci) [inline .github/workflows/multi_service_image_build.yml]: Looks like an error here. Need use parameters  <GIT_ORGANIZATION_NAME> & <WL_GITOPS_REPO_NAME>
As for me, need to replace git@github.com with some new parameter to have support for GitLab in the future.

File: .github/workflows/multi_service_image_build.yml [hunk 2]
```diff
@@ -54,32 +58,53 @@ jobs:
...
+                    DOCKERFILE="Dockerfile"
+                    BUILD_CONTEXT="${wl_service_dir}"
+              fi
               argo submit .argo/wl-service-wf.yaml \
                     --generate-name ${wl_service}-wf- \
-                    -p repo=git@github.com:<GIT_ORGANIZATION_NAME>/<WL_REPO_NAME>.git \
+                    -p repo=git@github.com:cgdevx-demo/vt-cnask.git \
                     -p wl-service-name=$wl_service \
-                    -p workload-name=<WL_REPO_NAME> \
+                    -p workload-name=vt-cnask \
```
Comment (sergs-pci) [inline .github/workflows/multi_service_image_build.yml]: workload name hardcoded, need replace to parameter

File: .github/workflows/multi_service_image_build.yml [hunk 3]
```diff
@@ -54,32 +58,53 @@ jobs:
...
+                    BUILD_CONTEXT=""
+                    echo "APP_STRUCT=1">>"$GITHUB_ENV"
+                else
+                    DOCKERFILE="Dockerfile"
+                    BUILD_CONTEXT="${wl_service_dir}"
+              fi
               argo submit .argo/wl-service-wf.yaml \
                     --generate-name ${wl_service}-wf- \
-                    -p repo=git@github.com:<GIT_ORGANIZATION_NAME>/<WL_REPO_NAME>.git \
+                    -p repo=git@github.com:cgdevx-demo/vt-cnask.git \
```
Comment (sergs-pci) [inline .github/workflows/multi_service_image_build.yml]: hardcoded values, need replace to parameters, as for me, need to replace git@github.com with some new parameter to have support for GitLab in the future.

File: .github/workflows/multi_service_image_build.yml (hunk 1 above)
Comment (sergs-pci) [reply .github/workflows/multi_service_image_build.yml]: probably <WL_GITOPS_REPOSITORY_GIT_URL> will work better here

File: .github/workflows/multi_service_image_build.yml [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
 on:
   push:
-    tags: ['[0-9]+.[0-9]+.[0-9]+-?*']
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+-?*'
 env:
   TAG: ${{ github.ref }}
   REPO_NAME: ${{ github.repository }}
-  ARGO_NAMESPACE: argo
+  ARGO_NAMESPACE: wl-vt-cnask-dev
```
Comment (sergs-pci) [inline .github/workflows/multi_service_image_build.yml]: Looks like an error here. Need use parameter. Something like that wl-<WL_NAME>-dev

**Review by sergs-pci — CHANGES_REQUESTED** [review]

File: .github/workflows/multi_service_image_build.yml (hunk 1 above)
Comment (vtsarfin-cg) [reply .github/workflows/multi_service_image_build.yml]: Fixed, thank you.

File: .github/workflows/multi_service_image_build.yml (hunk 2 above)
Comment (vtsarfin-cg) [reply .github/workflows/multi_service_image_build.yml]: Fixed, thank you.

File: .github/workflows/multi_service_image_build.yml (hunk 3 above)
Comment (vtsarfin-cg) [reply .github/workflows/multi_service_image_build.yml]: Fixed, thank you.

File: .github/workflows/multi_service_image_build.yml (hunk 4 above)
Comment (vtsarfin-cg) [reply .github/workflows/multi_service_image_build.yml]: Fixed, thank you.

**Review by sergs-pci — APPROVED** [review]

**Pull request merged**